src/app/ubs-admin/components/ubs-admin-order-payment/ubs-admin-order-payment.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="container">
+  <h1 class="my-4 form__headline">Оплата замовлення</h1>
+  <div class="row align-items-baseline">
+    <div class="col-md-4 col-xl-3">
+      <span class="form__subtitle">Замовлення виконано у розмірі</span>
+    </div>
+    <div class="col-md-2">
+      <span class="price">300 грн</span>
+    </div>
+    <div class="col-md-3 offset-md-1">
+      <button class="btn">+ Додати оплату</button>
+    </div>
+  </div>
+  <p class="need-to-pay">Потрібно доплатити 250 грн</p>
+  <div class="table-responsive-md col-md-10 pl-0">
+    <table class="table table-borderless">
```

Comment:
Add "<th>" headers to this "<table>".
Add a description to this table.

Reply:
We don't have a description to this table on Figma template

Reply:
use your imagination